SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
+
+Following Skosmos versions are currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.15    |  ✔️                 |
```

Comment:
I think having to maintain a table like this is a recipe for outdated information. Would it be better just to state that the latest vX.X-maintenance branch is supported and nothing else?